File: spec/bots/invalid-json-bot/scraper.rb
require 'json'

0.upto(9) do |n|
  record = {
    :licence_number => "XYZ#{n}",
    :source_url => 'http://example.com',
    :sample_date => '2014-06-01'
  }
  record = 'This is not JSON' if n == 5
  puts(record.to_json)
end

File: spec/bots/python-bot/scraper.py
from __future__ import print_function

import json

for n in range(10):
    record = {
      'licence_number': 'XYZ{}'.format(n),
      'source_url': 'http://example.com',
      'sample_date': '2014-06-01'
    }
    print(json.dumps(record))
